examples/context/motivation-problem.js

```diff
@@ -0,0 +1,23 @@
+const ThemedButton = props => {
+  //highlight-range{1}
+  return <Button theme={props.theme} />;
+};
+
+// An intermediate component
+const Toolbar = props => {
```

Comment:
We try to use the functional declaration syntax for functional components since it's familiar to more people. Would you mind doing that too?

```js
function Toolbar(props) {
  // ..
}
```